src/librustdoc/html/static/main.js

```diff
@@ -473,11 +473,6 @@ function hideThemeButtonState() {
             }
 
             if (h3) {
-                var collapses = h3.getElementsByClassName("collapse-toggle");
-                if (collapses.length > 0) {
-                    // The element is not visible, we need to make it appear!
-                    collapseDocs(collapses[0], "show");
-                }
```

Comment:
This section can now be combined with the next section, and the `if (h3)` removed. We expand details in the same way regardless of what type of element we're targetting.

Reply:
Indeed, cleaning it.